src/Microsoft.ML.DataView/VBuffer.cs

```diff
@@ -197,13 +246,23 @@ public void CopyTo(ref VBuffer<T> dst, int srcMin, int length)
         }
 
         /// <summary>
-        /// Copy from this buffer to the given destination array. This "densifies".
+        /// Copy from this buffer to the given destination span. This "densifies."
         /// </summary>
+        /// <param name="dst">The destination buffer. This <see cref="Span{T}.Length"/> must have least <see cref="Length"/>.</param>
         public void CopyTo(Span<T> dst)
         {
             CopyTo(dst, 0);
         }
 
+        /// <summary>
+        /// Copy from this buffer to the given destination span, starting at the specified index. This "densifies."
+        /// </summary>
+        /// <param name="dst">The destination buffer. This <see cref="Span{T}.Length"/> must be at least <see cref="Length"/>
+        /// plus <paramref name="ivDst"/>.</param>
+        /// <param name="ivDst">The starting index of <paramref name="dst"/> at which to start copying</param>
+        /// <param name="defaultValue">The value to fill in for the implicit sparse entries. This is a potential exception to
+        /// general expectation of sparse <see cref="VBuffer{T}"/> that the implicit sparse entries have the default value
+        /// of <typeparamref name="T"/>.</param>
         public void CopyTo(Span<T> dst, int ivDst, T defaultValue = default(T))
```

Comment:
>ivDst [](start = 44, length = 5)

Looks like we still have some acronyms and whatever this is lurking around. Not sure if care enough to change the API for v1...

Reply:
Note that changing a parameter name after you ship an API is a breaking change. So it's now or never.

Reply:
Well... I was hoping to not make this one of the cherry picked PRs. Let me do this, I'll do the review comments in one commit, then do the changes to parameters in an isolated commit, and then we can decide what we want to do.

Reply:
OK, so, last commit titled `Update parameter names to avoid abbreviations, e.g., src => source and suchlike.` is up, I updated the parameter names in `VBuffer`. Names in `VBufferEditor` were fine. I also updated the documenation in the `.md` file, but I did not update the actual code that inspired the example since that is in the internal API. (But, the documentation uses the idea merely as a simple, easy to understand example, so I think that's fine.) I took some queues from things like [Array.Copy](https://docs.microsoft.com/en-us/dotnet/api/system.array.copy?view=netstandard-2.0#System_Array_Copy_System_Array_System_Int32_System_Array_System_Int32_System_Int32_), but without going to far as calling the parameters `VBuffer<T> sourceBuffer` since I felt that might be going a bit too far, I just named them things like `source` rather than go quasi-hungarian w.r.t. the names, so, just `source`, `destination`, etc., as I believe you also did in `VBufferEditor` @eerhardt (e.g., your `Create` methods take `destination`, not `destinationBuffer`).

Reply:
BTW on the same subject of renamings, there was one place @eerhardt pointed out where an index was being named `slot` for some reason (since I guess we conceptually separate the concepts of "columns" and "slots" in data view, but, for VBuffer specifically, this should have just been `index`).